packages/sku/scripts/init.js

```diff
@@ -193,7 +193,7 @@ const getTemplateFileDestinationFromRoot =
       const destination = getTemplateFileDestination(file);
 
       // Ensure folders exist before writing files to them
-      await fs.mkdirp(path.dirname(destination));
+      await fs.ensureDir(path.dirname(destination));
```

Comment:
does `ensureDir` offer any value over native `mkdir`?

Since [Node v12](https://nodejs.org/api/fs.html#fspromisesmkdirpath-options) (possibly earlier) mkdir with recursive true will create any dir if it doesn't exist, and won't error if it does exist.

Reply:
Swapped to using `mkdir`, and consequently removed the `fs-extra` dep from sku entirely 🎉